MEAnalysis/crab_vhbb/multicrab.py

```diff
@@ -1,84 +1,169 @@
+datasets = {
+    'ttHTobb': {
+        "ds": '/ttHTobb_M125_13TeV_powheg_pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM',
+        "maxlumis": -1,
+        "perjob": 10,
+        "runtime": 40,
+        "mem_cfg": me_cfgs["default"],
+    },
+    'ttHToNonbb': {
+        "ds": '/ttHToNonbb_M125_13TeV_powheg_pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM',
+        "maxlumis": -1,
+        "perjob": 100,
+        "runtime": 40,
+        "mem_cfg": me_cfgs["default"],
+    },
+    'TTbar_inc': {
+        "ds": '/TT_TuneCUETP8M1_13TeV-powheg-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12_ext3-v1/MINIAODSIM',
+        "maxlumis": -1,
+        "perjob": 100,
+        "runtime": 40,
+        "mem_cfg": me_cfgs["default"],
+    },
+    'TTbar_sl': (
+        '/TTToSemiLeptonic_13TeV-powheg/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12_ext1-v1/MINIAODSIM',
+        -1, 100, 40, me_cfgs["leptonic"],
+    ),
+    'TTbar_dl': (
+        '/TTTo2L2Nu_13TeV-powheg/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12_ext1-v1/MINIAODSIM',
+        -1, 200, 40, me_cfgs["leptonic"],
+    ),
+    'QCD300': (
+        '/QCD_HT300to500_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM',
+        -1, 500, 40, me_cfgs["hadronic"],
+    ),
+    'QCD500': (
+        '/QCD_HT500to700_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM',
+        -1, 500, 40, me_cfgs["hadronic"],
+    ),
+    'QCD700': (
+        '/QCD_HT700to1000_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM',
+        -1, 300, 40, me_cfgs["hadronic"],
+    ),
+    'QCD1000': (
+        '/QCD_HT1000to1500_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM',
+        -1, 200, 40, me_cfgs["hadronic"],
+    ),
+    'QCD1500': (
+        '/QCD_HT1500to2000_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM',
+        -1, 100, 40, me_cfgs["hadronic"],
+    ),
+    'QCD2000': (
+        '/QCD_HT2000toInf_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM',
+        -1, 100, 40, me_cfgs["hadronic"],
+    ),
 }
 
-version = '_v2'  #***************CHANGE HERE***************
+#now we construct the workflows from all the base datasets
+workflow_datasets = {}
+workflow_datasets["leptonic"] = {}
+for k in ["ttHTobb", "ttHToNonbb", "TTbar_inc", "TTbar_sl", "TTbar_dl"]:
+    workflow_datasets["leptonic"][k] = datasets[k]
+
+workflow_datasets["hadronic"] = {}
+for k in datasets.keys():
+    if "QCD" in k or "ttH" in k:
+        workflow_datasets["hadronic"][k] = datasets[k]
```

Comment:
I think you can also achieve this by copying the tth dataset element with a new me_conf, e.g.

~~~
workflow_datasets["hadronic_tests"] = {
  "tth1": {... "mem_cfg": "cfg_FH1.py"},
  "tth2": {... "mem_cfg": "cfg_FH2.py"},
}
~~~